runtime-config-linux.md

```diff
@@ -17,7 +17,7 @@ The following parameters can be specified to setup namespaces:
     * **`uts`** the container will be able to have its own hostname and domain name
     * **`user`** the container will be able to remap user and group IDs from the host to local users and groups within the container
 
-* **`path`** *(string, optional)* - path to namespace file
+* **`path`** *(string, optional)* - path to namespace file in the host mount namespace
```

Comment:
On Fri, Dec 18, 2015 at 11:28:29AM -0800, Mrunal Patel wrote:
> > -* **`path`** *(string, optional)* - path to namespace file
> > +* **`path`** *(string, optional)* - path to namespace file in the host mount namespace
> 
> I think it is good the way it is. The path to a namespace file is
> unambiguous :)

Is that “we can close this PR” or “we don't need the ‘host’ →
‘runtime’ change [1]”?

[1]: https://github.com/opencontainers/specs/pull/275/files#r48057211